src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/repository/UserCourtRepository.kt

```diff
@@ -0,0 +1,13 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.repository
+
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Query
+import org.springframework.data.repository.query.Param
+import org.springframework.stereotype.Repository
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.entity.UserCourt
+
+@Repository
+interface UserCourtRepository : JpaRepository<UserCourt, Long> {
+  @Query(value = "from UserCourt uc where uc.username = :username")
```

Comment:
Do you need this query on here?  Won't Spring to it for you?

Reply:
No! It steadfastly refused to offer me the findAllByUsername option :-(